tests/scalers/new-relic.test.ts

```diff
@@ -0,0 +1,257 @@
+/*
+To use this test you will need:
+* NewRelic License Key
+* NewRelic API Key
+
+You can get a free account on https://www.newrelic.com/
+
+once you have your license and api key you need to setup the following
+environment variables
+
+NEWRELIC_API_KEY
+NEWRELIC_LICENSE
+NEWRELIC_ACCOUNT_ID
+
+the API key starts with 'NRAK' and the license ends in 'NRAL'
+
+ */
+import * as fs from 'fs'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import test from 'ava'
+
+const newRelicApiKey = process.env['NEWRELIC_API_KEY']
+const newRelicAccountId = process.env['NEWRELIC_ACCOUNT_ID']
+const testNamespace = 'new-relic-test'
+const newRelicNamespace = 'new-relic'
+const newRelicRepoUrl = 'https://helm-charts.newrelic.com'
+const newRelicRepoName = 'new-relic'
+const newRelicHelmPackageName = 'newrelic/nri-bundle'
+const newRelicLicenseKey = process.env['NEWRELIC_LICENSE']
+const kuberneteClusterName = 'keda-new-relic'
+
+
+test.before(t => {
+  if (!newRelicApiKey) {
+    t.fail('NEWRELIC_API_KEY environment variable is required for newrelic tests tests')
+  }
+  if (!newRelicLicenseKey) {
+    t.fail('NEWRELIC_LICENSE environment variable is required for newrelic tests tests')
+  }
+  if (!newRelicAccountId) {
+    t.fail('NEWRELIC_ACCOUNT_ID environment variable is required for newrelic tests tests')
+  }
+  sh.exec(`kubectl create namespace ${newRelicNamespace}`)
+  sh.exec(`helm repo add ${newRelicRepoName} ${newRelicRepoUrl}`)
+  sh.exec(`helm repo update`)
+  let helmInstallStatus = sh.exec(`helm upgrade --install --set global.cluster=${kuberneteClusterName} --set prometheus.enabled=true --set ksm.enabled=true --set global.lowDataMode=true --set global.licenseKey=${newRelicLicenseKey} --timeout 600s --set logging.enabled=false --set ksm.enabled=true --set logging.enabled=true --namespace ${newRelicNamespace} nri-keda ${newRelicHelmPackageName}`).code
```

Comment:
```suggestion
  let helmInstallStatus = sh.exec(`helm upgrade --install --set global.cluster=${kuberneteClusterName} --set prometheus.enabled=true --set ksm.enabled=true --set global.lowDataMode=true --set global.licenseKey=${newRelicLicenseKey} --timeout 600s --set logging.enabled=false --set ksm.enabled=true --set logging.enabled=true --namespace ${newRelicNamespace} nri-keda ${newRelicRepoName}/${newRelicHelmPackageName}`).code
```

Reply:
Fixed! thanks I was not seeing that error and was a 'dummy' error :D